virtualenvwrapper.el

```diff
@@ -94,12 +94,16 @@ to activate when one of them is found."
   "If a venv in the projetile root exists, activates it.
 Set your common venvs names in `venv-dirlookup-names'"
   (let ((path (--first
-                (file-exists-p it)
-                (--map (concat (projectile-project-root) it)
-                        venv-dirlookup-names))))
+               (file-exists-p it)
+               (--map (concat (projectile-project-root) it)
+                      venv-dirlookup-names))))
     (when path
-      (setq venv-current-name path) ;; there's really nothing that feels good to do here ;_;
-      (venv--activate-dir path))))
+      (if (and path (file-regular-p path) (file-readable-p path))
```

Comment:
Added a comment.
Please check if the comment makes it clear what the code does.